handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,72 @@
+# Engineering Manager - Backend Platform
+
+We are looking for an engineering manager to join the team that is building the backend platform upon which product engineering teams can iterate quickly and safely. The team's mission is to promote and champion consistency and simplicity across all backend code by building shared tools, libraries and infrastructure for common use cases, scale and maintain said infrastructure and ultimately create leverage for and accelerate other teams.
+
+The [Cloud team](https://about.sourcegraph.com/handbook/engineering/cloud) team will become the Backend Platform team once we hire a Product Manager to drive the Cloud project forward.
```

Comment:
```suggestion
This team is currently called our "Cloud" team because it is focused on [building and launching the first version of Sourcegraph Cloud](../cloud/goals.md#private-code-on-sourcegraph-cloud). In the short term, you will help the team balance work on Sourcegraph Cloud as well as engineering infrastructure. In the medium/long term, you will figure out how we transition to a state where we have a dedicated backend platform team while ensuring that our cloud product is sufficiently owned and maintained.
```